src/AM.Condo.Abstractions/IO/IGitRepositoryFactory.cs

```diff
@@ -24,13 +24,16 @@ public interface IGitRepositoryFactory
         /// <param name="uri">
         /// The URI of the git repository that should be cloned.
         /// </param>
+        /// <param name="authorization">
+        /// The authorization header to use for access the repository.
```

Comment:
*to access

Reply:
*to access

Reply:
rookie mistake @dmccaffery

Reply:
I changed it to accessing because you use present participles elsewhere.

Reply:
@sjk07 we can't all be winners :feelsgood: 

@tiffanywang3 fixed